bokehjs/src/coffee/embed.ts

```diff
@@ -48,6 +49,11 @@ declare interface Jupyter {
 
 declare var Jupyter: Jupyter | undefined
 
+export const kernels: {[key: string]: any} = {}
```

Comment:
Declare an interface for `Kernel` type, the same as it was done for `Comm` and `Jupyter`.